backend/uclapi/requirements.txt

```diff
@@ -41,6 +41,7 @@ raven==6.10.0
 redis==3.0.1
 requests==2.21.0
 requests-file==1.4.3
+requests-futures==0.9.9
```

Comment:
No problem with adding a requirement. But isn't there a previous one that we can remove now?

Reply:
Yes, we can remove grequests i believe, let me modify this list